### PR TITLE
Issue fixed in calender overlaping ui

### PR DESCRIPTION
**Pull request by vijay-webkul**
## Calender overlapping 
#6989 

-> Calennder Overlapping which is showing in sale on the Booking page issue has been fixed.

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@vijay-webkul 
It is working fine. [Here](https://tinyurl.com/2zm49ghc)

File: packages/Webkul/BookingProduct/src/Resources/views/admin/sales/bookings/index.blade.php [hunk 1]
```diff
@@ -11,6 +11,11 @@
         .grid-container .datagrid-filters .filter-right {
             grid-template-columns: auto auto auto;
         }
+        @media only screen and (max-width: 768px){
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/BookingProduct/src/Resources/views/admin/sales/bookings/index.blade.php:14]: Add new line after }

File: packages/Webkul/BookingProduct/src/Resources/views/admin/sales/bookings/index.blade.php [hunk 2]
```diff
@@ -11,6 +11,11 @@
...
+        .vuecal__no-event {
+            padding-top: 0rem!important;
+        }
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/BookingProduct/src/Resources/views/admin/sales/bookings/index.blade.php:17]: Add space  before  }

**Review by Sanjaybhattwebkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/BookingProduct/src/Resources/views/admin/sales/bookings/index.blade.php (hunk 1 above)
Comment (vijay-webkul) [reply packages/Webkul/BookingProduct/src/Resources/views/admin/sales/bookings/index.blade.php]: done

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
Now, this is **working fine.** please find the **screenshot** [Here](https://webkul.chatwhizz.com/share/screenshot/6426b95f83d74a61f22a4c2d)

**Review by Sanjaybhattwebkul — DISMISSED** [review]

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**